src/app/core/filter/filter/filter.component.spec.ts

```diff
@@ -152,37 +150,4 @@ describe("FilterComponent", () => {
     expect(component.filterSelections[0].name).toBe("category");
     expect(component.filterSelections[0].selectedOptionValues).toBeEmpty();
   });
-
```

Comment:
As we are using basicautocompletecomponent now, we not need this anymore?

Reply:
yes, I think the autocomplete-component only replaces the UI part but we still rely on other code to find and create the available options and to create the correct filter object, which is checked in the expect() statements here. Let's keep it